lib/usdUfe/ufe/UsdHierarchy.cpp

```diff
@@ -257,6 +257,19 @@ UsdHierarchy::insertChildCmd(const Ufe::SceneItem::Ptr& child, const Ufe::SceneI
     return UsdUndoInsertChildCommand::create(fItem, downcast(child), downcast(pos));
 }
 
+Ufe::InsertChildCommand::Ptr
+UsdHierarchy::appendChildVerifyRestrictionsCmd(const Ufe::SceneItem::Ptr& child)
+{
+    const auto childCmd = appendChildCmd(child);
+
+    if (childCmd && isConnected(downcast(child))) {
+        throw std::runtime_error("The node you're trying to move has connections.");
```

Comment:
We should update this message according to the updated and approved UI linked in the related story